<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShipDesigner</class>
 <widget class="QWidget" name="ShipDesigner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="itemList">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>5</y>
     <width>211</width>
     <height>401</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QListView" name="componentList">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>25</y>
     <width>151</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="componentFilterEdit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>0</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>&lt;Filter&gt;; </string>
   </property>
  </widget>
  <widget class="QFrame" name="componentInfoFrame">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>405</y>
     <width>351</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="componentName">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Component Info</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="componentDesc">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'FSAlbert'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>161</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="missionCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>205</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string comment="A">A (Assault)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="B">B (Battle Line)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="D">D (Drone)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="E">E (Escort)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="G">G (Missile)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="H">H (Heavy Screen)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="K">K (Orbital Command Group)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="L">L (Light Screen)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="N">N (Non Combatants)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="O">O (Auxiliary)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="P">P (PDC)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="S">S (Standoff)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="T">T (Transport)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="V">V (Carrier)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="X">X (Special #1)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="Y">Y (Special #2)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string comment="Z">Z (Special #3)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="designName_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Design Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="savedDesigns">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Saved Designs</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>155</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Starship</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Orbital Installation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Surface Fortress</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="designNameEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="designName_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mission Class</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QComboBox" name="designsCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="designName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Design Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>249</y>
     <width>161</width>
     <height>318</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>160</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Orders</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateNUD">
       <property name="text">
        <string>Generate NUD</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Quantity</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="quantityEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Pop Group</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="popGroupEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Priority #</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="priorityEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Fleet #</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="fleetEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="generateSHIP">
       <property name="text">
        <string>Generate SHIP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateBI">
       <property name="text">
        <string>Generate BIs</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>5</y>
     <width>271</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Weight:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="tonnageLabel">
        <property name="text">
         <string>0 tons</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Action Pts:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Warp Bubble</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string># Items</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="countLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Construction Costs</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="detailedTable"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>0</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="application.qrc">
     <normaloff>:/icons/arrow-left.png</normaloff>:/icons/arrow-left.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
